front_end_pocket_poll/lib/categories_widgets/category_create.dart

```diff
@@ -51,7 +51,7 @@ class _CategoryCreateState extends State<CategoryCreate> {
     this.nextChoiceValue = 2;
```

Comment:
This needs to change to 1 now that we (finally) are starting at index 0.

Reply:
I mean it technically works but you’re right it should be 1 :)